packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/JsonModelCoreTests.cs

```diff
@@ -82,5 +82,17 @@ public void CamelCaseSerializedName()
             var file = writer.Write();
             Assert.AreEqual(Helpers.GetExpectedFromFile(), file.Content);
         }
+
+        [Test]
+        public void MultipleAdditionalProperties()
```

Comment:
Can we also add a test where there is an explicit AdditionalProperties of BinaryData?

Reply:
done